Kucoin.Net/SubClients/KucoinClientSpot.cs

```diff
@@ -1164,6 +1164,81 @@ public async Task<WebCallResult<IEnumerable<KucoinOrder>>> GetStopOrderByClientO
             return await Execute<IEnumerable<KucoinOrder>>(GetUri("stop-order/queryOrderByClientOid"), HttpMethod.Get, ct, parameters, signed: true).ConfigureAwait(false);
         }
 
+        #region Margin Trade
+
+        #region Borrow & Lend
+
+        /// <summary>
+        /// Places a Borrow order (https://docs.kucoin.com/#post-borrow-order)
+        /// </summary>
+        /// <param name="currency">Currency to Borrow e.g USDT etc</param>
+        /// <param name="type">The type of the order (FOK, IOC)</param>
+        /// <param name="quantity">Total size</param>
+        /// <param name="maxRate">The max interest rate. All interest rates are acceptable if this field is left empty</param>
+        /// <param name="term">erm (Unit: Day). All terms are acceptable if this field is left empty. Please note to separate the terms via comma. For example, 7,14,28</param>
+        /// <param name="ct">Cancellation token</param>
+        /// <returns>The id of the new order</returns>
+        public async Task<WebCallResult<KucoinNewBorrowOrder>> PlaceBorrowOrderAsync(
+            string currency,
+            KucoinBorrowOrderType type,
+            decimal quantity,
+            decimal? maxRate = null,
+            string? term = null,
+            CancellationToken ct = default)
+        {
+
+            var parameters = new Dictionary<string, object>
+            {
+                { "currency", currency },
+                { "type", JsonConvert.SerializeObject(type, new BorrowOrderTypeConverter(false)) },
+                { "size", quantity }
+            };
+            parameters.AddOptionalParameter("maxRate", maxRate);
+            parameters.AddOptionalParameter("term", term);
+            return await Execute<KucoinNewBorrowOrder>(GetUri("margin/borrow"), HttpMethod.Post, ct, parameters, true).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Get info on a specific borrow order (https://docs.kucoin.com/#get-borrow-order)
+        /// </summary>
+        /// <param name="clientOrderId">The client order id of the borrow order</param>
+        /// <param name="ct">Cancellation token</param>
+        /// <returns>Borrow Order info</returns>
+        public async Task<WebCallResult<KucoinBorrowOrder>> GetBorrowOrderByClientOrderIdAsync(string clientOrderId, CancellationToken ct = default)
```

Comment:
I don't think this is actually by `clientOrderId`. By `clientOrderId` would be when you can retrieve the order by an id you passed in when placing the order. This is a generated id, so this should just be called `GetBorrowOrderAsync` and the parameter should be `orderId`